rsc/application.properties.dist

```diff
@@ -13,6 +13,9 @@ server.servlet.session.cookie.http-only=true
 spring.pid.fail-on-write-error=true
 spring.pid.file=OH_API_PID
 
+### In production change to http://<domain>
+cors.allowed.origins=http://localhost:8080,http://localhost:3000
```

Comment:
Is it possible to use env vars like in `api.host=API_HOST:API_PORT`?

Reply:
good catch, thanks!